Cargo.toml

```diff
@@ -13,19 +13,34 @@ categories = ["concurrency", "wasm"]
 readme = "README.md"
 
 [features]
+default = ["es_modules"]
```

Comment:
This is the only problem I had with the PR, I needed to update { default-features = false }

Reply:
I thought module support is now common enough to be default. I will bump the version of course.